src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/sessiondays/SessionDaysReport.java

```diff
@@ -218,10 +219,12 @@ private void setReportDetail(SessionDaysCaseData caseData, List<SessionDaysRepor
                         reportDetail.setHearingDate(LocalDateTime.parse(
                                 dateListedTypeItem.getValue().getListedDate(), OLD_DATE_TIME_PATTERN)
                                 .toLocalDate().toString());
+                        String judgeName = getJudgeName(hearingTypeItem.getValue().getJudge());
```

Comment:
Line 154 and 222 seem the same.

Reply:
yeah in two different methods to get judge Name for different purposes. In 154 judgeName is needed to find out his employment status. In 222, for saving it in reportDetail's property.